tests/unit/builders.makePurl.spec.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+import { PackageURL } from 'packageurl-js'
+import { DummyComponent, BomBuilder } from '../../dist/builders'
+
+const urlQualifiers = {
+  vcs_url: 'git+https://github.com/CycloneDX/cyclonedx-npm.git',
+  download_url: 'https://registry.npmjs.org/@cyclonedx/cyclonedx-npm/-/cyclonedx-npm-1.0.0.tgz'
```

Comment:
:x: add more qualifiers, some tat should not be stripped, to a test may show that they re untouched.